### ISSUE TITLE
Track: error while tracking objects

### ISSUE THREAD

**Issue by Sharan-Sundar** [dlstreamer/dlstreamer#265]
Hi, 
I am currently performing vehicle detection using a custom-trained ssd model. 
This is using `OpenVINO 2021.4.2 LTS` on Ubuntu `18.04.6 LTS`  with the latest DLstreamer release - #254 .
The output of  `cat /proc/cpuinfo | grep "model name"`  is     `model name	: Intel(R) Core(TM) i7-9700 CPU @ 3.00GHz`.
Gstreamer version is `GStreamer Core Library version 1.18.4 `.

This is my pipeline -
`gst-launch-1.0 filesrc location=/home/user/Downloads/vid.mp4 ! decodebin !    videoconvert n-threads=4  ! capsfilter caps="video/x-raw,format=BGRx" !   gvadetect model=ir_weights/VT.xml  ! gvatrack tracking-type=short-term !  gvawatermark name=gvawatermark !    fakesink sync=false`

This is the output I get -

```
Setting pipeline to PAUSED ...
Pipeline is PREROLLING ...
Got context from element 'vaapipostproc0': gst.vaapi.Display=context, gst.vaapi.Display=(GstObject)"\(GstVaapiDisplayGLX\)\ vaapidisplayglx0";
Redistribute latency...
INFO:
/root/gst-video-analytics/gst/elements/gvatrack/gstgvatrack.c(264): gst_gva_track_set_caps (): /GstPipeline:pipeline0/GstGvaTrack:gvatrack0
Pipeline is PREROLLED ...0 %)
Setting pipeline to PLAYING ...
New clock: GstSystemClock
ERROR: from element /GstPipeline:pipeline0/GstGvaTrack:gvatrack0: transform_ip failed
Additional debug info:
/root/gst-video-analytics/gst/elements/gvatrack/gstgvatrack.c(296): gst_gva_track_transform_ip (): /GstPipeline:pipeline0/GstGvaTrack:gvatrack0:

Track: error while tracking objects
Execution ended after 0:00:11.878513694
Setting pipeline to NULL ...
Freeing pipeline ...

```
It seems to happen randomly - I am not able to point out a particular time instant in the video where this happens. The pipeline ends at different times in different runs. I went through #224 as well but there you had concluded that you weren't able to reproduce the issue. Any idea where the issue might be ?

**Comment by howarwang** [comment]
Did you specify the model-proc of the gvadetect?

**Comment by Sharan-Sundar** [comment]
No I did not specify as I still have not understood model-proc properly. Anyway I don't need pre/post processing so I assumed it wasn't needed.

But, on seeing your comment, I tried but that led to no detections from gvadetect at all. The detection model is ssd-mobilenet and it's detecting only one object class - "Vehicle". This is the model-proc I used - 
```
{
    "json_schema_version": "2.1.0",
    "input_preproc": [],
    "output_postproc": [
        {
            "converter": "tensor_to_bbox_ssd",
            "labels": [
                "Vehicle"
            ]
        }
    ]
}
```
Am I using it correctly? Kindly help me out here.
